tests/subsys/fs/fs_api/src/test_multi_fs.c

```diff
@@ -69,6 +74,11 @@ static int test_fs_readmount(void)
 
 static int test_fs_deinit(void)
 {
+	/* NULL parameter */
+	if (fs_unregister(TEST_FS_1, NULL) == 0) {
+		return TC_FAIL;
```

Comment:
Use here `-EINVAL` as in all other cases; or convert all other cases to `TC_FAIL`.